auditlog/admin.py

```diff
@@ -22,10 +30,12 @@ class LogEntryAdmin(admin.ModelAdmin, LogEntryAdminMixin):
         "timestamp",
         "object_repr",
         "changes",
-        "actor__first_name",
-        "actor__last_name",
-        f"actor__{get_user_model().USERNAME_FIELD}",
-    ]
+        f"actor__{user_model.USERNAME_FIELD}",
+    ] + (
```

Comment:
Something like the following adds whichever field is available, handling the case where either `fist_name` or `last_name` do actually exist

```
    search_fields = [
        "timestamp",
        "object_repr",
        "changes",
        f"actor__{get_user_model().USERNAME_FIELD}",
    ] + [f"actor__{f.name}" for f in get_user_model()._meta.get_fields() if f.name in ["first_name", "last_name"]]
```

Reply:
This is better because includes whatever fields that are available.
Feel free to create a new PR with tests and changelog